Documentation/upgrades/upgrading-etcd.md

```diff
@@ -0,0 +1,19 @@
+# Upgrading etcd clusters and applications
+
+This section contains documents specific to upgrading etcd clusters and applications.
+
+Moving from etcd API v2 to API v3
```

Comment:
Also agreed. We don't have any of these sorts of pages yet. Part of this exercise will be defining the format.